src/cca/Microsoft.Identity.Client.Confidential/Microsoft.Identity.Client.Confidential.csproj

```diff
@@ -0,0 +1,87 @@
+<Project Sdk="MSBuild.Sdk.Extras">
+
+  <PropertyGroup>
+    <TargetFrameworkNetDesktop>net45</TargetFrameworkNetDesktop>    <!--TODO: needed? -->
+    <TargetFrameworkNetStandard>netstandard1.3</TargetFrameworkNetStandard>
+    <TargetFrameworkNetCore>netcoreapp2.1</TargetFrameworkNetCore>
+
+    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Windows'))">$(TargetFrameworkNetDesktop);$(TargetFrameworkNetStandard);$(TargetFrameworkNetCore);</TargetFrameworks>
+    <TargetFrameworks Condition="!$([MSBuild]::IsOsPlatform('Windows'))">$(TargetFrameworkNetStandard);$(TargetFrameworkNetCore);</TargetFrameworks>
+
+    <PathToSources>$(MSBuildThisFileDirectory)</PathToSources>
+  </PropertyGroup>
+
+  <PropertyGroup Label="NuGet and AssemblyInfo metadata">
```

Comment:
@jmprieur - you might want to review the package metadata (this is for the confidential client standalone nuget)